cli/cli_test.go

```diff
@@ -39,7 +39,8 @@ func TestCmdMarshalProto(t *testing.T) {
  "sharedv4": false,
  "queue_depth": 0,
  "force_unsupported_fs_type": false,
- "nodiscard": false
+ "nodiscard": false,
+ "storage_policy": ""
```

Comment:
Is this supposed to be equal to https://github.com/libopenstorage/openstorage/pull/857/files#diff-604f4192deea5adf9c159e5f88f64630R59 ?

Reply:
that's json name if you see `"force_unsupported_fs_type"` at https://github.com/libopenstorage/openstorage/pull/857/files#diff-604f4192deea5adf9c159e5f88f64630R58 
it's defined here api.pb.go  https://github.com/libopenstorage/openstorage/blob/5a0f8125e94f6b045d5a702cce1669c51e704585/api/api.pb.go#L1534